taxi/views.py

```diff
@@ -85,6 +111,20 @@
     model = Driver
```

Comment:
The `DriverListView` model attribute is correctly set. Ensure that the corresponding template uses the `object_list` or `driver_list` context variable.